translations/de.ts

```diff
@@ -85,7 +86,7 @@ WARNING: slider is not supposed to work yet.</source>
     <message>
         <location filename="../src/widget/form/settings/avsettings.ui" line="54"/>
         <source>Filter sound from your microphone, so that people hearing you would get better sound.</source>
-        <translation>Die Filterung verbessert deine Sprachqualität.</translation>
+        <translation>Das Filtern verbessert die Tonqualität.</translation>
```

Comment:
as it says "from your mic", I suppose it is either a noise filter or an echo filter.